models/library.go

```diff
@@ -0,0 +1,10 @@
+package models
+
+import (
+	"github.com/knqyf263/go-dep-parser/pkg/types"
+)
+
+type LibraryScanner struct {
```

Comment:
__[Golint]__ exported type LibraryScanner should have comment or be unexported ([link](https://sider.review/gh/repos/54808920/pulls/829#issue-4176674))

<sub>posted by Sider</sub>